Interdomain Routing FRR/BGP Transit AS/kathara-lab_bgp-transit-as-redistribute-bgp-non-deterministic_frr/as10rt2/etc/frr/frr.conf

```diff
@@ -0,0 +1,48 @@
+! as10rt2
+!
+! FRRouting configuration file
+!
+password zebra
+enable password zebra
+!
+log file /var/log/frr/frr.log
+!
+! RIP CONFIGURATION
+!
+router rip
+network eth1
+redistribute connected
+redistribute bgp route-map eBGP
+!
+route-map eBGP permit 10
+match ip next-hop address 12.0.0.2
+!
+! BGP CONFIGURATION
+!
+router bgp 10
+no bgp network import-check
```

Comment:
There are no announced prefixes that are not already in the FIB, so no need for this command